### ISSUE TITLE
Unable to send email.

### ISSUE THREAD

**Issue by Geethatnt** [Geethatnt/Bulkmail#1]
Bulkmail is to send emails to all those who subscribed for the newsletter. Once i run this code i could not able to send mails.

**Comment by SajanRajah** [comment]
Is the issue still open? I can assist to resolve.

**Comment by Geethatnt** [comment]
Yes Thank you . Please review Bulkmailtest.php in this repository.

**Comment by SajanRajah** [comment]
Line 29 in Bulkmailtest.php
Change the port to 587 from 465. 587 is for ecrypted TLS setting whereas 465 is for SSL. These are not interchangeable.
